content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -264,8 +276,8 @@ click on a test file named `spec-a.js` via
 executes the files in the following order:
 
 ```html
-<!-- bundled support file -->
-<script src="support/index.js"></script>
+<!-- bundled e2e or component support file -->
```

Comment:
Can you use the `:::e2e-component-example` directive wherever you've got "e2e or component" to create tabbed examples?

Also, the script source should probably start with `e2e` and not `integration` and look more like `e2e/spec-a.cy.js`. Also, component spec paths might look more like `components/Button/Button.cy.js`